docs/performance_data.md

```diff
@@ -0,0 +1,21 @@
+Performance Data
+================
+
+awx has the ability to collect performance data on job runs.
+
+The following data is collected periodically (by default, every 0.25 seconds):
+* cpu usage
```

Comment:
Capitalize to `CPU`

Reply:
... and `Memory` on the line below

Reply:
love it, thank you! our docs are going to ✨